tests/helpers.py

```diff
@@ -9,13 +9,19 @@
 
 import psycopg2
 from loguru import logger
+from pydantic import BaseModel
 from sqlalchemy.engine.url import make_url
 
 from lnbits import core
 from lnbits.db import DB_TYPE, POSTGRES
 from lnbits.wallets import get_wallet_class, set_wallet_class
 
 
+class TestModel(BaseModel):
```

Comment:
nitpick:
```suggestion
class TestPerson(BaseModel):
```

Reply:
i disagree, it is meant as a generic model not a person